xonsh/ptk/shell.py

```diff
@@ -53,17 +54,19 @@ def cmdloop(self, intro=None):
                 completions_display = builtins.__xonsh_env__.get('COMPLETIONS_DISPLAY')
                 multicolumn = (completions_display == 'multi')
                 completer = None if completions_display == 'none' else self.pt_completer
-                line = prompt(
-                    mouse_support=mouse_support,
-                    auto_suggest=auto_suggest,
-                    get_prompt_tokens=token_func,
-                    style=style_cls,
-                    completer=completer,
-                    lexer=PygmentsLexer(XonshLexer),
-                    history=self.history,
-                    enable_history_search=True,
-                    key_bindings_registry=self.key_bindings_manager.registry,
-                    display_completions_in_columns=multicolumn)
+                with self.prompter:
+                    line = self.prompter.prompt(
+                        mouse_support=mouse_support,
+                        auto_suggest=auto_suggest,
+                        get_prompt_tokens=token_func,
+                        style=style_cls,
+                        completer=completer,
+                        lexer=PygmentsLexer(XonshLexer),
+                        history=self.history,
+                        enable_history_search=True,
+                        reserve_space_for_menu=0,
+                        key_bindings_registry=self.key_bindings_manager.registry,
```

Comment:
I have a change on the `xonfig` branch that fixes this, so maybe we can wait for this one.